lib/backend/k8s/resources/networkpolicy.go

```diff
@@ -237,6 +237,8 @@ func (c *networkPolicyClient) List(ctx context.Context, list model.ListInterface
 	}
 
 	// List all Namespaced Calico Network Policies.
+	// For consistency, we should split the revision that was input into CRD and K8s revisions,
+	// but the CRD client will fail on any non-empty revision anyway, so don't bother.
```

Comment:
> but the CRD client will fail on any non-empty revision anyway

Is this just because the `List` operation doesn't support revisions at all?

Reply:
Yeah, exactly.  customresource.go line 245 has this to say

```
	if revision != "" {
		return nil, errors.New("Cannot List this resource type specifying a ResourceVersion")
	}
```